File: test/cs/test_PriestlyTaylor.cs
'Test generation'

public static class Test
{
    //first);
    //test1

    public static void test1()
    {
        PriestlyTaylor res0 = Estimation_PriestlyTaylor.CalculatePriestlyTaylor(Alpha:1.5,psychrometricConstant:0.66,hslope:0.584,netRadiationEquivalentEvaporation:638.142);

        Console.WriteLine("{0}","evapoTranspirationPriestlyTaylor: "+res0.evapoTranspirationPriestlyTaylor);

        Console.WriteLine("evapoTranspirationPriestlyTaylor Comparison: ("+Math.Round(449.367,3)+";"+Math.Round(res0.evapoTranspirationPriestlyTaylor,3)+") "+Equals(Math.Round(449.367,3),Math.Round(res0.evapoTranspirationPriestlyTaylor,3)));
    }
}

File: test/java/test_PriestlyTaylor.java
//Test generation'

import java.math.BigDecimal;
public static class Test
{
    //first);
    //test1);

    public static void test1()
    {
        double netRadiationEquivalentEvaporation = 638.142;
        double hslope = 0.584;
        double psychrometricConstant = 0.66;
        double Alpha = 1.5;

        PriestlyTaylor res0 = Estimation_PriestlyTaylor.CalculatePriestlyTaylor(netRadiationEquivalentEvaporation,hslope,psychrometricConstant,Alpha);
        System.out.println(" evapoTranspirationPriestlyTaylor: "+res0.evapoTranspirationPriestlyTaylor);

        System.out.println(((new BigDecimal(res0.evapoTranspirationPriestlyTaylor)).setScale(3, BigDecimal.ROUND_HALF_DOWN)).equals((new BigDecimal(449.367)).setScale(3, BigDecimal.ROUND_HALF_DOWN)));
    }
}
